lte/gateway/c/core/oai/common/security_types.h

```diff
@@ -122,8 +129,18 @@ typedef struct eutran_vector_s {
   res_t xres;
   uint8_t autn[AUTN_LENGTH_OCTETS];
   uint8_t kasme[KASME_LENGTH_OCTETS];
+  uint8_t ck[CK_LENGTH_OCTETS];
+  uint8_t ik[IK_LENGTH_OCTETS];
```

Comment:
I don't think you need these any more...

Reply:
Comment addressed.